etna/models/base.py

```diff
@@ -166,6 +107,36 @@ def context_size(self) -> int:
     def forecast(self, ts: TSDataset) -> TSDataset:
```

Comment:
Haven't made attention before. Why does mypy not complain about signatures (LSP issue)?